### PR TITLE
More v1 improvements

### PR DESCRIPTION
**Pull request by notkevin888**
* More resources
* Descriptions to learning list
* More Professional About Us (tm)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -8,8 +8,8 @@
 <h1>About</h1>
 
 <p>
-  About us: We're just a couple of Square-1 enthusiasts who wanted to learn how to code
-  our own website.
+  About us: We're a couple of Square-1 enthusiasts who wanted to show others the elegance and efficiency of OBL
+  with our expertise in creating websites.
```
Comment (notkevin888) [inline about.html]: hm, maybe this is too extra lol

Comment (sharikak54) [reply about.html]: yeah, kind of too extra especially since you have no *expertise* in creating websites :D

**Review by sharikak54 — APPROVED** [review]
I made a couple adjustments so it looks nicer, but yeah seems good.

**Comment by sharikak54** [comment]
I poked around cubegym and it doesn't really seem finished.  Do we want to link the whole thing yet?  (Mostly since the PBL section is just a picture of a solved puzzle...)

**Comment by notkevin888** [comment]
> I poked around cubegym and it doesn't really seem finished. Do we want to link the whole thing yet? (Mostly since the PBL section is just a picture of a solved puzzle...)

Ooh, didn't realize the PBL trainer wasn't built out yet - yeah, let's remove that

File: about.html [hunk 2]
```diff
@@ -8,8 +8,8 @@
 <h1>About</h1>
 
 <p>
-  About us: We're just a couple of Square-1 enthusiasts who wanted to learn how to code
-  our own website.
+  We're a couple of Square-1 enthusiasts who wanted to show others the elegance and efficiency of OBL
+  with our expertise in creating websites.
```
Comment (sharikak54) [inline about.html]:
```suggestion
  We're just a couple of Square-1 enthusiasts who wanted to show others the elegance and efficiency of OBL.
```

**Pull request merged**